src/index.html

```diff
@@ -22,6 +22,483 @@
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--laptops"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <a
+            href="#"
+            class="card__image"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+              class="card__img"
+            />
+          </a>
+        </div>
+
+        <div class="card__bottom">
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <span class="card__code">Product code: 195434</span>
+
+          <div class="card__quality">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <span class="card__reviews">Reviews: 5</span>
+          </div>
+
+          <div class="card__price">
+            <span class="card__text">Price:</span>
+            <span class="card__value">$2,199</span>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+      <div
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div
```